scripts/auth_tools.py

```diff
@@ -4,15 +4,21 @@
 from constants import URL
 import requests
 
-secret = (
-    open(os.path.dirname(os.path.abspath(__file__)) + "/.secret", "r")
-    .read()
-    .strip("\n")
-)
-user = (
-    open(os.path.dirname(os.path.abspath(__file__)) + "/.user", "r").read().strip("\n")
-)
 
+def create_secret_file(file_name: str, value: str):
+    with open(os.path.dirname(os.path.abspath(__file__)) + f"/.{file_name}", "w") as f:
+        f.write(value)
+
+def read_secret_file(file_name: str) -> str:
```

Comment:
`create/read_credentials_file` would be a better name as this sounds very much like `.secret ` file